tests/lib/cli.js

```diff
@@ -1380,6 +1380,84 @@ describe("cli", () => {
         });
     });
 
+    describe("--report-unused-disable-directives", () => {
```

Comment:
Since these options are available in both modes (eslintrc and flat config), can we run these tests in both modes? (for example, move this to line 1370 and use `useFlatConfig` variable instead of `true`).